lib/score-me.js

```diff
@@ -1,29 +0,0 @@
-const got = require('got')
-const nullScoreMeData = require('./null-score-me-data')
-
-class ScoreMe {
-  constructor (url) {
-    this.jsonUrl =
-      `http://readme-score-api.herokuapp.com/score.json?url=${url}&force=true`
```

Comment:
Making this API endpoint as environment configuration, could help in case user want to use different service.

Reply:
Great catch, @sairampooraj, thanks for the review.

I'll work on that now and notify you when done.

Reply:
@sairampooraj I've moved that value out of the source code. Please see [`.env.example`](https://github.com/commonality/readme-inspector/pull/4/files#diff-84d28cbdd7fa7039305534ae1a842f9c), as well as the [README's installation instructions](https://github.com/commonality/readme-inspector/tree/GH-1-feat-inspect-readmes#1-installation) for details.

Thanks again!

Reply:
@sairampooraj everything's passing. Mind reviewing again?